src/HavenoDaemon.test.ts

```diff
@@ -465,6 +465,102 @@ test("Can manage Monero daemon connections", async () => {
   if (err) throw err;
 });
 
+test("Can start and stop local Monero node", async() => {
+  let rootDir = process.cwd();
+  let localNodeUrl: string = "http://127.0.0.1:38081"; // local node connection generated by test
```

Comment:
Why does the test generate a connection to http://127.0.0.1:38081? Perhaps it's done automatically when starting the daemon process with startup flags? 

In that case, I think it makes sense to switch CoreMoneroConnectionsService and CoreMoneroNodeService to hardcode "127.0.0.1" instead of "localhost", since they're the same, and then the test will not create a new connection.

Reply:
Correct, when a MoneroDaemonRpc is created as a local process will automatically generate a daemon that has the loopback address 127.0.0.1 which is added to CoreMoneroConnectionsService. I set the current connection to the local daemon since the other classes in Haveno call into CoreMoneroConnectionsService for access to the monero daemon.

If the different CoreMoneroConnectionsService and CoreMoneroNodeService are decoupled I would not have this problem.

Reply:
Removed